src/navigation/MainTabNavigator.tsx

```diff
@@ -79,9 +85,8 @@ const MainTabNavigator: FunctionComponent = () => {
   }) => {
     return (
       <SvgXml
-        xml={
-          focused ? Icons.SelfAssessmentActive : Icons.SelfAssessmentInactive
-        }
+        xml={TabBarIcons.CheckInBox}
+        fill={determineIconFill(focused)}
```

Comment:
Oooh now I see that you extracted this function since it's called multiple times. One option might be to have a base `TabIcon` component in this file that always accepts a `fill` prop and handles the badge and fill logic, then have your `SelfAssessmentIcon` render that component. 

This is up to your discretion, though, I think it's fine as is!

Reply:
Oh I see that's a good idea. I'm going to try that and will tag you in a commit.

Reply:
Thoughts on 5b8303c?

Reply:
Oops I mean 933998f29abe11a8eacc8cd0cc73a2d0dea921cb!

Reply:
@devinjameson yesss looks great 👏

Reply:
Thanks!